src/Entity/CommerceContentEntityBase.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Drupal\commerce\Entity;
+
+use Drupal\Core\Entity\ContentEntityBase;
+use Drupal\Core\Language\LanguageInterface;
+
+/**
+ * Provides the base class for Commerce content entities.
+ */
+class CommerceContentEntityBase extends ContentEntityBase implements CommerceContentEntityInterface {
```

Comment:
i think the usage could be extented on other place - for example any getStore(s) needs
and thanks